src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/Resources/etag.feature

```diff
@@ -125,14 +133,22 @@ Feature: ETag validations
                       "type": "urn:ed-fi:api:optimistic-lock-failed",
                       "title": "Optimistic Lock Failed",
                       "status": 412,
-                      "correlationId": null,
+                      "validationErrors": {},
                       "errors": [
                           "The resource item's etag value does not match what was specified in the 'If-Match' request header indicating that it has been modified by another client since it was last retrieved."
                       ]
                   }
                   """
 
-        @ignore @API-264
-        Scenario: 06 Ensure that clients can pass an ETag to delete a resource
+        Scenario: 05 Ensure that clients can pass an ETag to delete a resource
+            Given a POST request is made to "/ed-fi/students" with
+                  """
+                  {
+                      "studentUniqueId": "111111",
+                      "birthDate": "2014-08-14",
+                      "firstName": "Russella",
+                      "lastSurname": "Mayers"
+                  }
+                  """
              When a DELETE if-match "{etag}" request is made to "/ed-fi/students/{id}"
```

Comment:
{etag}  - I don't think this variable now in code to replace it . It should be {IfMatch}